aws/aws-cdk
# fix(cli): CLI ignores profile in cdk.json

validation steps
* Prepare that the command like `cdk diff $SOME_STACK_NAME --profile $SOME_PROFILE_NAME` succeeds
* Execute `cdk diff $SOME_STACK_NAME` and receive failures to confirm no credential is implicitly provided.
* Add "profile": "$SOME_PROFILE_NAME" to cdk.json. Execute `cdk diff $SOME_STACK_NAME` and it must finish successfully.

### Commit Message
fix(cli): CLI ignores profile in cdk.json (#7398)

The profile option was not being passed from cdk.json.

Get the values from the merged configuration, instead of using the profile argument directly.

Closes #3007
### End Commit Message

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*
